docs/code/MlNetCookBook.md

```diff
@@ -293,11 +288,11 @@ In the file above, the last column (12th) is label that we predict, and all the
 ```csharp
 // Create a new environment for ML.NET operations. It can be used for exception tracking and logging, 
```

Comment:
>environment [](start = 16, length = 11)

Would be a good idea to replace with the above comments:

// Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
// as a catalog of available operations and as the source of randomness.